### ISSUE TITLE
scala 2.13 artifact

### ISSUE THREAD

**Issue by tyrantkhan** [kolotaev/ride#8]
would you consider releasing a version of this for scala 2.13?

**Comment by kolotaev** [comment]
Hi, yes. I've already thought about it. Expect it to be done soon.

**Comment by tyrantkhan** [comment]
awesome! I'm moving a project from go to scala and plan on using 2.13. Would be great to use this because I used rs/xid to generate user_ids in the first version :) .

Much appreciated!

**Comment by kolotaev** [comment]
I've released a new artifact for Scala 2.13. It has a version of 1.1.1.
But due to the sync problems on Bintray it hasn't been synced to JCenter. I've reached their support, but have no estimate when the issue will be resolved. So, meanwhile you can use this resolver that directly points to Bintray:

```scala
resolvers += Resolver.bintrayRepo("kolotaev", "maven")

libraryDependencies ++= Seq(
  "com.github.kolotaev" %% "ride" % "1.1.1"
)
```

**Comment by tyrantkhan** [comment]
awesome! Really appreciate the quick turnaround!
